File: loops/fizzbuzz.c
// Governor-Nephi Bakare
#include <stdio.h>
#include <math.h>

int skibidi;

int main(void){
    for(skibidi+1;skibidi<=50;skibidi++){
        if (skibidi % 5 == 0 && skibidi % 3 == 0){
        printf("FizzBuzz\n");
    }
        else if (skibidi % 5 == 0){
        printf("Buzz\n");
    }
         else if (skibidi % 3 == 0){
        printf("Fizz\n");
    }  
    
        else{
        printf("%d\n", skibidi);
    }   
    }
}

File: loops/fizzbuzz.py
# Governor-Nephi Bakare, FizzBuzz Python

numero = 1

while numero <=50:
    if numero % 5 == 0 and numero % 3 == 0:
        print("FizzBuzz")
    elif numero % 5 == 0:
        print("Buzz")
    elif numero % 3 == 0:
        print("Fizz")
    else:
        print(numero)
    numero+=1